raizama/core/tests/test_command_import_spreadsheet.py

```diff
@@ -0,0 +1,219 @@
+from io import StringIO
+from pathlib import Path
+from django.core.management import call_command
+from django.test import TestCase
+from raizama.core.models import Organization
+from raizama.settings import BASE_DIR
+
+
+class ImportSpreadSheetTest(TestCase):
+    def setUp(self):
+        self.csv_path = Path(
+            BASE_DIR,
+            "raizama",
+            "core",
+            "tests",
+            "fixtures",
+            "OrganizacaoEndEContatos-CasosDeTeste.csv",
+        )
```

Comment:
`BASE_DIR` já é um `Path`, então isso fica mais simples assim:

```suggestion
        self.csv_path = BASE_DIR /  "raizama" / "core" / "tests" / "fixtures" / "OrganizacaoEndEContatos-CasosDeTeste.csv"
```

Talvez a linha fique muito longa, mas aei tu pode organizar de forma diferente (por exemplo, ter uma constante com o diretório de _fixtures_).